bridgehead

```diff
@@ -64,7 +64,7 @@ loadVars() {
 	fetchVarsFromVaultByFile /etc/bridgehead/$PROJECT.conf || fail_and_report 1 "Unable to fetchVarsFromVaultByFile"
 	setHostname
 	optimizeBlazeMemoryUsage
-	[ -e ./$PROJECT/vars ] && source ./$PROJECT/vars
+	[ -e ./$PROJECT/vars ] && source ./$PROJECT/vars ./vars
```

Comment:
I think the source of ./vars should happen before the more specific project vars and regardless of project vars existing or not